modules/lib/test/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+        "lib": ["DOM"]
```

Comment:
Is this correct. This is server side code we are looking at, so the DOM probably shouldn't be available (even for the tests).

Reply:
I probably did that because I was mocking log with console, but since I've removed all console mocks and I'm only testing types, not actually running the code, there is no need for that, so I've removed the file entirely.